app_feup/lib/redux/ActionCreators.dart

```diff
@@ -207,12 +285,33 @@ ThunkAction<AppState> getUserFees(Completer<Null> action) {
       var response = await NetworkRouter.getWithCookies(url, query, cookies);
 
       String feesBalance = await parseFeesBalance(response);
-      store.dispatch(new SetFeesBalanceAction(feesBalance));
-
       String feesLimit = await parseFeesNextLimit(response);
+
+      String current_time = DateTime.now().toString();
+      Tuple2<String, String> userPersistentInfo = await AppSharedPreferences.getPersistentUserInfo();
+      if(userPersistentInfo.item1 != "" && userPersistentInfo.item2 != ""){
+
+        // Store refresh time
+        AppRefreshTimesDatabase refreshTimesDatabase = await AppRefreshTimesDatabase();
+        await refreshTimesDatabase.saveRefreshTime("fees", current_time);
```

Comment:
Since both user fees and print balance have their refresh time saved the same way, couldn't this be extracted to simplify code? ThunkActions are becoming big methods so anything we can generalize is probably a good approach!

Reply:
Extracted to a method.